### PR TITLE
Notifications with db

### PR DESCRIPTION
**Pull request by AlexFry123**

### CODE REVIEW

File: src/app/domains/Notification/components/list/NotificationList/NotificationList.template.js [hunk 1]
```diff
@@ -1,55 +1,57 @@
...
+      .collection('notifications')
+      .where('userId', '==', md5(session.email))
   )
-}
 
-NotificationList.propTypes = {
-  collectionName: PropTypes.string.isRequired
+  // USE EFFECTS
+  useEffect(() => {
+    getData('users', md5(session.email)).then((result) => {
```
Comment (GingerBoy14) [inline src/app/domains/Notification/components/list/NotificationList/NotificationList.template.js:23]: Use COLLECTIONS.USERS

File: src/app/domains/Notification/components/list/NotificationList/NotificationList.template.js [hunk 2]
```diff
@@ -1,55 +1,57 @@
 import { NotificationAdvancedView } from '../../views/NotificationAdvancedView'
```
Comment (okrdima) [inline src/app/domains/Notification/components/list/NotificationList/NotificationList.template.js:1]: fix import

File: src/app/domains/Notification/components/list/NotificationList/NotificationList.template.js [hunk 3]
```diff
@@ -1,55 +1,57 @@
...
-    notificationTime: new Date()
-  }
-]
+import { useState, useEffect } from 'react'
+import { firestore, getData } from 'app/services/Firestore'
+import { useCollection } from 'react-firebase-hooks/firestore'
+import { useSession } from 'app/context/SessionContext/hooks'
+import md5 from 'md5'
 
 //TODO delete mock data
```
Comment (okrdima) [inline src/app/domains/Notification/components/list/NotificationList/NotificationList.template.js:8]: delete comment

File: src/app/domains/Notification/components/views/NotificationSimpleView/NotificationSimpleView.template.js [hunk 4]
```diff
@@ -1,22 +1,44 @@
...
-      </Badge>
-    </IconButton>
+  // CUSTOM HOOKS
+  const session = useSession()
+  const history = useHistory()
+  const [value, loading, error] = useCollection(
+    firestore
+      .collection('notifications')
+      .where('userId', '==', md5(session.email))
   )
```
Comment (GingerBoy14) [inline src/app/domains/Notification/components/views/NotificationSimpleView/NotificationSimpleView.template.js:22]: Use can use getCollectionRef instead firestore.collection()

File: src/app/domains/Notification/components/list/NotificationList/NotificationList.template.js [hunk 5]
```diff
@@ -1,55 +1,57 @@
...
+const NotificationList = (props) => {
+  // STATE
+  const [data, setData] = useState([])
+
+  // CUSTOM HOOKS
+  const session = useSession()
+  const [value, loading, error] = useCollection(
+    firestore
+      .collection('notifications')
+      .where('userId', '==', md5(session.email))
```
Comment (GingerBoy14) [inline src/app/domains/Notification/components/list/NotificationList/NotificationList.template.js:18]: Same here

**Review by GingerBoy14 — CHANGES_REQUESTED** [review]

**Review by okrdima — APPROVED** [review]

File: src/app/components/Header/Header.template.js [hunk 6]
```diff
@@ -1,21 +1,22 @@
...
 import { useStyles } from './Header.style'
+import { NotificationSimpleView } from 'domains/Notification/components/views'
 import { auth } from 'app/services/Auth'
+
 const Header = (props) => {
+  // INTERFACE
   const { goBack } = props
+
+  // CUSTOM HOOKS
   let history = useHistory()
```
Comment (5P4C3GR33D) [inline src/app/components/Header/Header.template.js:16]: It should be 'const history', because it's not changeable variable, but it's not necessary.

**Review by 5P4C3GR33D — APPROVED** [review]

**Review by GingerBoy14 — APPROVED** [review]

**Pull request merged**